interfaces/HO-Portal/src/app/services/programs-service-api.service.ts

```diff
@@ -220,15 +220,14 @@ export class ProgramsServiceApiService {
     password: string,
     role: string,
     status: string,
-    countryId: number,
   ): Promise<any> {
     return this.apiService
       .post(environment.url_121_service_api, `/user`, {
         email,
         password,
         role,
         status,
-        countryId,
+        countryId: 1,
```

Comment:
@elwinschmitz did you leave this in here, because this can only be removed once the back-end is changed as well?